apps/miranum-cli/src/app/deployment/deployment.ts

```diff
@@ -19,13 +19,13 @@ export class Deployment {
                 case "forms":
                     // eslint-disable-next-line @typescript-eslint/ban-ts-comment
                     // @ts-ignore
-                    (await getFiles(`${path}/${workspace[key]}`.replace("//", "/"), [".form", ".schema.json"]))
+                    (await getFiles(`${path}/${workspace[key]}`.replace("//", "/"), [this.digiwfLib.projectConfig?.workspace.forms.extension]))
```

Comment:
Der Workspace war bisher als type `object` in der MiranumConfig definiert.  Bisher war das ja nur ein einfaches Json Object.
Jetzt geben wir eine ganze Struktur für den Workspace vor. Dementsprechend sollte der Type des Workspaces auch geändert werden (bzw. es macht Sinn einen richtigen Typen dafür zu definieren).